src/utils/axios.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+import { SERVER_URL } from '../constants'
+
+const axiosInstance = axios.create({
+  baseURL: SERVER_URL,
```

Comment:
Can you test removing the `SERVER_URL` from the services that already exist? I think we just need the endpoint now